### ISSUE TITLE
Search for differences between website and designs

### ISSUE THREAD

**Issue by alexdor** [ossn/fixme#28]
Design images can be found [here](https://github.com/ossn/fixme/tree/master/screens)

**Comment by Timileyin5** [comment]
Can you elaborate on what needs to be done here?

**Comment by alexdor** [comment]
@Timileyin5 you are right, the description is a bit vague. The final website needs to be checked for inconsistencies with the original designs. 

Does this make more sense?

**Comment by alexdor** [comment]
A,B,C are all different states of the homepage, for example in A the "I do my magic in" part is empty, in B the user has selected a few languages and in C the user has selected a few languages and a few more options. 

So you all of them are the right one : )

**Comment by Timileyin5** [comment]
okay, that makes sense. Thank you for explaining!

**Comment by Dhruv34788** [comment]
@alexdor In screenshots of the design footer contains various links to connect like email, GitHub but it is not reflected in the code let me know if I can move ahead to fix this.

**Comment by alexdor** [comment]
@bacharakis do we have all the links so we can add them to the footer?

**Comment by Jay341997** [comment]
@alexdor In image Mozilla_Fixme_Screen02_Issues.png there is a select button for sort by. so can you tell me what types are there for sort by?

**Comment by alexdor** [comment]
I'm not sure what sorting types should be included to be honest, for sure the issue date (ascending or descending) is going to be on of them. @bacharakis any other order by filters?

**Comment by Muljayan** [comment]
### White line on the right side of the learn more button 
![image](https://user-images.githubusercontent.com/37667437/47055614-e3823280-d1d5-11e8-831a-a06b6c66c6ae.png)
> Browser : Google chrome 69.0.3497.100 (Official Build) (64-bit)
> O/S: Ubuntu 18.04
